minecraft-server/README.md

```diff
@@ -267,28 +269,111 @@ downloaded, scaled, and converted from any other image format:
     docker run -d -e ICON=http://..../some/image.png ...
 
 ### Rcon
+
 To use rcon use the `ENABLE_RCON` and `RCON_PASSORD` variables.
+By default rcon port will be `25575` but can easily be changed with the `RCON_PORT` variable.
```

Comment:
I wonder if it's better to recommend they do a port mapping rather adjust the config...but then you already have the code for that so doesn't need to change.